### ISSUE TITLE
Some Full Screen Mode Problems

### ISSUE THREAD

**Issue by zo-shin** [segler-alex/RadioDroid#642]
1 - The history items can’t be deleted.
2 - Keep changing between previous and next, there will be some same items in history.
3 - Click the three-dot symbol at the right top, and there are one **share** command in the menu and another in **homepage**. At least one of them should be removed.
4 - Click any favorite or history item with no internet connection and **off-Wi-Fi warning** disabled, there will be an abnormal playing time (maybe it’s the current time incorrectly shown) keeping running at the left bottom of album art.
5 - There is unknown time shown between playing time and bit rate, which doesn’t run all the time.
6 - If there is no lyrics-supported app installed, the **lyrics** command won’t respond. There should be a notification or the command should be invisible.
7 - Suggestion: album art can be shown automatically without scrolling down.

**Comment by werman** [comment]
> 1 - The history items can’t be deleted.

Yes, would be better for them to be removable, however don't know by which action. Maybe add "remove all" for now...

> 2 - Keep changing between previous and next, there will be some same items in history.

I'm looking only at the last history item. On the other hand it may be worth not to add items with length of several seconds.

> 3 - Click the three-dot symbol at the right top, and there are one share command in the menu and another in homepage. At least one of them should be removed.

Probably missed it.

> 4 - Click any favorite or history item with no internet connection and off-Wi-Fi warning disabled, there will be an abnormal playing time (maybe it’s the current time incorrectly shown) keeping running at the left bottom of album art.
> 5 - There is unknown time shown between playing time and bit rate, which doesn’t run all the time.

It is buffer time - the time which could be played without internet. However I agree it's not that clear. I just didn't want empty space there.

> 6 - If there is no lyrics-supported app installed, the lyrics command won’t respond. There should be a notification or the command should be invisible.

Yes, there is even TODO in code =)

`// TODO: send to QuickLyric's download page`

> 7 - Suggestion: album art can be shown automatically without scrolling down.

Did you test in emulator, there are issues in emulator with scrolling of full screen player.

**Comment by zo-shin** [comment]
I didn’t find any problem with scrolling of full screen player in the emulator, but I found the app not compatible with Android 4.4.2, it crashed on it.
Abnormal playing time when there is no internet connection, which ends looking like current time:
![3](https://user-images.githubusercontent.com/38669548/71519405-60d52700-28f2-11ea-95d1-ea7929988f7d.jpg)
Stream info can’t be fully shown in bottom navigation
![1](https://user-images.githubusercontent.com/38669548/71519448-82cea980-28f2-11ea-8b0b-78c21ec56cf0.jpg)
Stream info can’t be correctly shown
![2](https://user-images.githubusercontent.com/38669548/71519462-91b55c00-28f2-11ea-8221-6003a31c7149.jpg)
